integration/src/test/java/dev/ikm/tinkar/integration/provider/ephemeral/EphemeralProtobufIT.java

```diff
@@ -23,37 +23,42 @@
 import dev.ikm.tinkar.entity.util.EntityProcessor;
 import dev.ikm.tinkar.entity.util.EntityRealizer;
 import dev.ikm.tinkar.integration.TestConstants;
+import dev.ikm.tinkar.integration.helper.DataStore;
 import dev.ikm.tinkar.integration.helper.TestHelper;
-import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestMethodOrder;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.File;
-import java.io.IOException;
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
-public class EphemeralProtobufIT extends TestHelper {
+public class EphemeralProtobufIT {
 
     private static final Logger LOG = LoggerFactory.getLogger(EphemeralProtobufIT.class);
 
     @BeforeAll
-    public void setupSuite() {
-       startEphemeralDataBase();
+    public static void beforeAll() {
```

Comment:
I don't think this needs to be static.

Reply:
@stenkarl, as discussed JUnit enforces static for this typeof TestInstance.Lifecycle configuration. We expect to reconsider this as we mature our testing architecture, especially as we reorganize integration tests and implement test suites.